### PR TITLE
Implement call sign search

### LINKED ISSUE #6: Call Sign Search
**Describe the feature you'd like**
Allow users to look up callsigns and return the data from [https://hamdb.org/api](https://hamdb.org/api)

### PR DESCRIPTION
**Pull request by nayabw**
Resolves #6

### CODE REVIEW

File: deploy-commands.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const { REST, Routes } = require('discord.js');
-const { clientId, guildId, token } = require('./secrets.json');
```
Comment (IanDuncanT) [inline deploy-commands.js]: This should show secrets.json

Comment (IanDuncanT) [reply deploy-commands.js]: Use soft linking to link secrets.json to secrets_5XB.json on development and secrets_W5AC.json on production

**Review by IanDuncanT — CHANGES_REQUESTED** [review]
Other than the one issue, it looks good

File: deploy-commands.js (hunk 1 above)
Comment (nayabw) [reply deploy-commands.js]: Missed that one

**Review by IanDuncanT — APPROVED** [review]

**Comment by IanDuncanT** [comment]
[Callsign Lookup](https://trello.com/c/hYbyL9NV/5-callsign-lookup)

**Pull request merged**